packages/inspection-capture-web/src/PhotoCapture/PhotoCapture.tsx

```diff
@@ -103,6 +105,11 @@ export function PhotoCapture({
   const onLastSightTaken = () => {
     startTasks()
       .then(() => {
+        analytics.trackEvent('Inspection completed', {
+          inspectionId,
+          category: 'inspection_completed',
```

Comment:
What does the `category` tag do exactly?

Reply:
It meant to be used as filter in our dasboards instead of adding all the different events one by one. 
For example, comparing the sum of 3 differents 'category' events versus 4 differents 'category' event.